src/screens/About.js

```diff
@@ -28,6 +29,16 @@ export class About extends React.Component {
     },
   }));
 
+  touchCount = 0;
+
+  onPressLogo = () => {
+    // Method created to test error handling
```

Comment:
Maybe you should set a timer to clear this counter after N seconds.

Reply:
Do you really think it's necessary? Every time the user leaves the 'About' screen, the count goes to 0 again. It's almost impossible to click 10 times in a row on the logo by mistake

Reply:
I didn't know it would reset every time user left the screen. I think it is ok.